server/src/test/java/org/opensearch/cluster/coordination/JoinTaskExecutorTests.java

```diff
@@ -922,16 +922,16 @@ private DiscoveryNode newDiscoveryNode(Map<String, String> attributes) {
         );
     }
 
-    private static final String SEGMENT_REPO = "segment-repo";
-    private static final String TRANSLOG_REPO = "translog-repo";
+    public static final String SEGMENT_REPO = "segment-repo";
```

Comment:
Nit: This should probably be in a constants file for tests. It looks like this is referenced across packages at the same level which is uncommon I think.